src/lib/y2partitioner/actions/controllers/encryption.rb

```diff
@@ -38,6 +39,12 @@ class Encryption < Base
         # @return [Symbol] :keep, :encrypt, :remove
         attr_accessor :action
 
+        # @return [Boolean] Whether the user wants to encrypt the device
+        attr_accessor :encrypt
```

Comment:
At first glance, I don't see where this is used. Is the value ever set?

Reply:
You are right. That value is controlled by the controller `Filesystem`.

Reply:
Seems to be a leftover from the migration from [`Filesystem` controller](https://github.com/yast/yast-storage-ng/blob/7892ca30d46c32206af69f4fe862a4cac39f645f/src/lib/y2partitioner/actions/controllers/filesystem.rb#L64). It's used by the [`FormatAndMount`](https://github.com/yast/yast-storage-ng/blob/80745be7874014ec769465018b212f856280f71e/src/lib/y2partitioner/widgets/format_and_mount.rb#L716) widget there, but it is not necessary here :) I'll remove it.

Thanks!